src/resource.jl

```diff
@@ -21,6 +21,9 @@ const resource_re = Regex("""
 """, "x")
 const hash_part_re = Regex("/($hash_re)\$")
 
+compress(io::IO) = TranscodingStream(GzipCompressor(level=9), io)
+decompress(io::IO) = TranscodingStream(GzipDecompressor(), io)
```

Comment:
No need to involve `TranscodingStreams`. `GzipDecompressorStream` can be used directly.

Reply:
Can you give an example of how that would be done? I've only used it via TranscodingStreams so far...

Reply:
See https://github.com/GunnarFarneback/LocalPackageServer.jl/blob/734490f1db4e0e1597fc262566c897931781b1c0/src/resource.jl#L115.

Without checking I'm fairly certain it's exactly the same under the hood, you can just avoid a direct dependency on TranscodingStreams.